app/routes/adapter.$name/route.tsx

```diff
@@ -46,11 +46,18 @@ export default function Index() {
 
     const adapter = filteredAdapters.length > 0 ? filteredAdapters[0] : null;
 
+    const displayName = (adapter: string) => {
```

Comment:
No need to send in adapter string. Adapter is accessible. Use it directly like adapter.name 😊